ui/component/or-rules/src/or-rule-validity.ts

```diff
@@ -225,6 +225,30 @@ export class OrRuleValidity extends translate(i18next)(LitElement) {
 
         this._dialog = showDialog({
             title: i18next.t("scheduleRuleActivity"),
+            styles: html`
+                <style>
+                    .mdc-dialog__surface {
+                        overflow-x: visible !important;
+                        overflow-y: visible !important;
+                    }
+
+                    #dialog-content {
+                        overflow: visible;
+                    }
+
+                    @media only screen and (max-width: 1279px) {
```

Comment:
Usually breakpoints in this area are at 1200. See for instance Bootstrap's media queries:
https://getbootstrap.com/docs/5.0/layout/breakpoints/#max-width